src/content/homepage.md

```diff
@@ -292,6 +292,15 @@ todoListSection:
           complete: false
         - text: Grant Program V2
           complete: false
+shopSection:
+  title: Get our swag
+  subtitle: Shard your life a little 🤘
```

Comment:
That's the only thing I am not sure of 😆

Reply:
Mouhahha

Reply:
@MaudSimon Can you take a look at the wording? Consider my text as a placeholder.